collect_app/src/main/java/org/odk/collect/android/utilities/CameraUtils.java

```diff
@@ -123,10 +124,8 @@ public static void savePhoto(String path, byte[] data) {
             Timber.e(e);
         }
     }
-  
-    public static String getVideoFilePath(Context context) {
-        final File dir = context.getExternalFilesDir(null);
-        return (dir == null ? "" : (dir.getAbsolutePath() + "/"))
-                + System.currentTimeMillis() + ".mp4";
+
+    public static String getVideoFilePath() {
+        return new StoragePathProvider().getTmpVideoPath();
```

Comment:
If you're going to be creating a new `StoragePathProvider` here anyway I think you can do away with the indirection and just use `new StoragePathProvider().getTmpVideoPath()` inline.

Reply:
Done.